### ISSUE TITLE
get metrics accuracy in compile

### ISSUE THREAD

**Issue by hoangtubatu123** [CyberZHG/keras-transformer#18]
When i compile the model in metrics, i can't fit the model with training data
